landing page/css/main.css

```diff
@@ -0,0 +1,76 @@
+@font-face {
+	font-family: hLight;
+	src: url(HelveticaNeue.ttf);
+}
+@font-face {
+	font-family: hBold;
+	src: url(HelveticaBold.ttf);	
+}
+body, html {
+	width: 100%;
+	overflow-x: hidden;
+	margin: 0px;
+	padding: 0px;
+	color: #fff;
+	font-family: hLight;
+	text-align: center;
+}
+span {
+	transition: all 0.5s ease;
+}
+#splash {
+	background: #ff2b06;
+	width: 10px;
+	height: 10px;
+	transform: translate(50vw, 50vh);
+	transform-origin: 50% 50%;
+	border-radius: 100%;
+}
+.hidden {
+	display: none;
+}
+small {
+	color: #fff !important;
+	margin-left: 20px;
+}
+ul {
+	list-style-type: none;
+	padding: 0px;
+}
+li {
+	display: inline;
+	margin: 10px;
+}
+#main_info_cont, #coming_soon {
+	opacity: 0;
+}
+.jumbotron {
+	min-height: 100vh;
+	background: transparent;
+}
+.transformation {
+	opacity: 0;
+}
+h1 {
+	font-family: hBold;
+}
+
+@media only screen and (max-width: 1000px) {
+    span {
+    	margin-left: -10px;
+    }
+}
+@media only screen and (max-width: 325px) {
+    #main_info_cont {
+    	margin-top: 50px !important; 
+    }
+    small {
+    	display: block;
+    }
+    ul {
+    	margin-top: 40px !important;	
+    }
+    footer {
+    	margin-top: 100px !important;	
+    }
+}
```

Comment:
Every file must contain a newline character at the end, in short, an empty line at the end.